services/bots/src/github-webhook/utils/teams.ts

```diff
@@ -0,0 +1,31 @@
+import { Organization } from '../github-webhook.const';
+import { GithubClient } from '../github-webhook.model';
+
+const ORG_TEAM_SEP = '/';
+const TEAM_PREFIX = `${Organization.HOME_ASSISTANT}${ORG_TEAM_SEP}`;
+
+export const expandTeams = async (
```

Comment:
I think we should extend [`WebhookContext`](https://github.com/home-assistant/service-hub/blob/main/services/bots/src/github-webhook/github-webhook.model.ts#L36) with this function and change the name to `expandOrganizationTeams`

Reply:
Extend was my initial thought, but if we for the most part pass it, we can continue with that here.